### PR TITLE
[DOCFIX] Add the outline for Alluxio contribution roadmap

### PR DESCRIPTION
**Pull request by LuQQiu**
Add the outline for Alluxio contribution roadmap

### CODE REVIEW

File: docs/en/roadmap/Introduction.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+- Prerequisites
+  - [Environment Setup]({{ '/en/roadmap/Environment-Setup.html' | relativize_url }})
+  - [Knowledge Required]({{ '/en/roadmap/Knowledge.html' | relativize_url }})
+- [Getting Started with Alluxio]({{ '/en/roadmap/Getting-Started-With-Alluxio.html' | relativize_url }})
+- [Getting Started with Alluxio contribution]({{ '/en/roadmap/Getting-Started-With-Alluxio-Contribution.html' | relativize_url }})
+
+## Motivations Behind the Contribution Roadmap
+
+### By [@LuQQiu](https://github.com/LuQQiu):
```
Comment (LuQQiu) [inline docs/en/roadmap/Introduction.md:34]: @HelloHorizon @yuzhu Recommended (optional) to add your words to make it not that formal and 打鸡血 to the new viewers :)

File: docs/en/roadmap/Introduction.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Catalog
+
+- Prerequisites
+  - [Environment Setup]({{ '/en/roadmap/Environment-Setup.html' | relativize_url }})
+  - [Knowledge Required]({{ '/en/roadmap/Knowledge.html' | relativize_url }})
+- [Getting Started with Alluxio]({{ '/en/roadmap/Getting-Started-With-Alluxio.html' | relativize_url }})
+- [Getting Started with Alluxio contribution]({{ '/en/roadmap/Getting-Started-With-Alluxio-Contribution.html' | relativize_url }})
+
+## Motivations Behind the Contribution Roadmap
```
Comment (LuQQiu) [inline docs/en/roadmap/Introduction.md:32]: In the future, can use another section for people to writing out their feelings and thoughts after they went through the roadmap.

File: docs/en/roadmap/Introduction.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+# Alluxio Contribution Roadmap (In Progress)
+
+We warmly welcome you to the Alluxio community. We are excited about your contributions and engagement with our project!
+This guide aims to give you step-by-step instructions on how to get started becoming a contributor to the Alluxio open source project
+and become a qualified independent contributor eventually.
+
+This guide is not a quick start guide but provides a comprehensive roadmap of how to grow from a person beginning with only passion and motivation 
+to a qualified independent contributor that can make useful contributions to open source projects like Alluxio.
+
+If you are a student, considering it as an internship with industry training.
```
Comment (Xenorith) [inline docs/en/roadmap/Introduction.md:16]: considering it as -> consider this to be

File: docs/en/roadmap/Introduction.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+We warmly welcome you to the Alluxio community. We are excited about your contributions and engagement with our project!
+This guide aims to give you step-by-step instructions on how to get started becoming a contributor to the Alluxio open source project
+and become a qualified independent contributor eventually.
+
+This guide is not a quick start guide but provides a comprehensive roadmap of how to grow from a person beginning with only passion and motivation 
+to a qualified independent contributor that can make useful contributions to open source projects like Alluxio.
+
+If you are a student, considering it as an internship with industry training. 
+If you are a new grad, considering it as the onboarding guide provided by your company to train you to be an independent contributor. 
+If you want to contribute to an open-source project and expand your capability zone, this guide helps you to expose to an industry-level project while learning java, git, distributed system, etc.
```
Comment (Xenorith) [inline docs/en/roadmap/Introduction.md:18]: this guide introduces you to an industry-level project while learning about java, git, distributed systems, and much more.

**Review by Xenorith — COMMENTED** [review]
i would suggest to target a feature branch until more parts are filled out to avoid publishing incomplete docs to the website

**Review by Xenorith — COMMENTED** [review]
to create a new category on the left sidebar, add it to `categoryList` list and `categoryInfo` map in `docs/_config.yml`

File: docs/en/roadmap/Introduction.md (hunk 1 above)
Comment (yuzhu) [reply docs/en/roadmap/Introduction.md:34]: Emmm, this seems great as a blog to go with the roadmap, but to include it in the git repo seems a bit strange..

Comment (LuQQiu) [reply docs/en/roadmap/Introduction.md:34]: yeah, that's true, then i remove it from now

**Comment by LuQQiu** [comment]
@Xenorith Addressed the comments, PTAL

File: docs/en/roadmap/Introduction.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+# Alluxio Contribution Roadmap (In Progress)
+
+We warmly welcome you to the Alluxio community. We are excited about your contributions and engagement with our project!
+This guide aims to give you step-by-step instructions on how to get started becoming a contributor to the Alluxio open source project
+and become a qualified independent contributor eventually.
+
+This guide is not a quick start guide but provides a comprehensive roadmap of how to grow from a person beginning with only passion and motivation 
+to a qualified independent contributor that can make useful contributions to open source projects like Alluxio.
+
+If you are a student, considering this to be an internship with industry training.
```
Comment (Xenorith) [inline docs/en/roadmap/Introduction.md:16]: considering -> consider and same for below

File: docs/en/roadmap/Environment-Setup.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+layout: global
+title: Environment Setup
+group: Roadmap
```
Comment (Xenorith) [inline docs/en/roadmap/Environment-Setup.md:4]: the group name must match one of the category names so this needs to be `Contribution Roadmap` and same for all the other files

**Review by Xenorith — CHANGES_REQUESTED** [review]

File: docs/en/roadmap/Introduction.md (hunk 5 above)
Comment (LuQQiu) [reply docs/en/roadmap/Introduction.md]: changed, thanks!

File: docs/en/roadmap/Environment-Setup.md (hunk 6 above)
Comment (LuQQiu) [reply docs/en/roadmap/Environment-Setup.md]: Changed, thanks!

File: docs/en/contribution-roadmap/Introduction.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+If you want to contribute to an open-source project and expand your capability zone, this guide introduces you to an industry-level project while learning about java, git, distributed system, and much more.
+If you are an experienced engineer who wants to understand Alluxio better, this guide helps you to speed up this process with resources that you can pick from.
+
+If you have any questions or suggestions regarding the contribution roadmap, feel free to share in the [Alluxio Community Slack Channel](https://slackin.alluxio.io).
+Many Alluxio experts in the channel may have gone through the same learning process as you are and are willing to answer your questions!
+
+## Catalog
+
+- Prerequisites
+  - [Environment Setup]({{ '/en/roadmap/Environment-Setup.html' | relativize_url }})
```
Comment (Xenorith) [inline docs/en/contribution-roadmap/Introduction.md:27]: urls are not updated after renaming the folder, says jenkins build

**Review by Xenorith — APPROVED** [review]
agree with david that the personal background section belongs more in a blog. i think you should still keep that content and when the roadmap is complete, create a new blog post in the alluxio.io site to introduce this new docs section

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
@yuzhu @HelloHorizon @ZhuTopher FYI, the roadmp outline is created in `contribution-roadmap` branch. Feel free to modify your section or add new docs if needed. Appreciate all the help!

**Pull request merged**